### PR TITLE
Run prettier

### PR DESCRIPTION
**Pull request by deshpsuy**
## Description
Ran prettier

## Motivation and Context
To make all code follow same coding style

## How Has This Been Tested?
All tests passing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
   "lint-staged": {
     "*.js": [
       "eslint --cache --fix",
-      "prettier --write"
+      "prettier --ignore-path .gitignore --write ."
```
Comment (deshpsuy) [inline package.json:114]: Updating prettier hook and ignore gitignore

File: .gitignore [hunk 2]
```diff
@@ -11,3 +11,6 @@ docs/api
 dist
 .tmp
 *.swp
+.idea
+.vscode
```
Comment (deshpsuy) [inline .gitignore:15]: ignored IntelliJ and vscode config files

File: package.json (hunk 1 above)
Comment (pradeepnschrodinger) [reply package.json:114]: So the prettier never worked because we didn't specify the path?

Comment (deshpsuy) [reply package.json:114]: yup

File: webpack.config.js [hunk 3]
```diff
@@ -7,40 +7,35 @@ var UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 
 var isDev = JSON.stringify(process.env.NODE_ENV !== 'production');
 
-var banner = (
+var banner =
   '/**\n' +
-  ' * FixedDataTable v' + packageJSON.version + ' \n' +
+  ' * FixedDataTable v' +
```
Comment (pradeepnschrodinger) [inline webpack.config.js:12]: supernit: although the output will be the same, I prefer the earlier format. I wonder if there're more such cases in other files.

Maybe we should have prettier ignore this section, or define this using template literals.

File: README.md [hunk 4]
```diff
@@ -52,91 +52,83 @@ Implementing a table involves three component types- `<Table/>`,`<Column/>`, and
...
-        {rows[rowIndex]}
-        </Cell>
-      )}
-      width={2000}
-    />
+const rows = [0, 1, 2];
+
+<Column
+  header={<Cell>Column 1</Cell>}
+  cell={({ rowIndex, ...props }) => <Cell {...props}>{rows[rowIndex]}</Cell>}
```
Comment (pradeepnschrodinger) [inline README.md:70]: For now, let's leave mark down files out of prettier's hands as some of the changes here seem subjective. 

We can have a separate PR to focus on these later.

File: README.md [hunk 5]
```diff
@@ -170,33 +161,28 @@ ReactDOM.render(
...
+## Contributions
 
 Use [GitHub issues](https://github.com/schrodinger/fixed-data-table-2/issues) for requests.
 
 We actively welcome pull requests; learn how to [contribute](https://github.com/schrodinger/fixed-data-table-2/blob/master/CONTRIBUTING.md).
 
 **BY CONTRIBUTING TO FIXEDDATATABLE, YOU AGREE THAT YOUR CONTRIBUTIONS WILL BE LICENSED UNDER THE BSD LICENSE [(see here)](https://github.com/schrodinger/fixed-data-table-2/blob/master/LICENSE)**. Furthermore, by contributing to FixedDataTable, you hereby grant to Schrödinger and any recipients of your contributions, including but not limited to users of this site, a perpetual, worldwide, non-exclusive, no-charge, royalty-free, irrevocable license to reproduce, prepare derivative works of, publicly display, publicly perform, sublicense, use, make and distribute your contributions and any derivative works under all intellectual property rights including but not limited to copyright and patent. **BY CONTRIBUTING TO FIXEDDATATABLE, YOU REPRESENT AND WARRANT THAT YOU ARE LEGALLY ENTITLED TO GRANT THE FOREGOING LICENSE**.
 
-Changelog
----------
```
Comment (pradeepnschrodinger) [inline README.md:194]: Is there any visual change here?

File: package.json (hunk 1 above)
Comment (wcjordan) [reply package.json:114]: I may also be worth ignoring the build_helpers & markdown files

File: package.json [hunk 6]
```diff
@@ -111,7 +111,7 @@
   "lint-staged": {
     "*.js": [
       "eslint --cache --fix",
-      "prettier --write"
+      "prettier --write {src,site,test,examples}/**/*.js"
```
Comment (deshpsuy) [inline package.json:114]: Only run prettier on js files present in these directories

File: .gitignore [hunk 7]
```diff
@@ -11,3 +11,5 @@ docs/api
 dist
 .tmp
 *.swp
+.eslintcache
+.idea
```
Comment (deshpsuy) [inline .gitignore:15]: Added to gitignore

File: package.json (hunk 1 above)
Comment (deshpsuy) [reply package.json]: Done

File: webpack.config.js (hunk 3 above)
Comment (deshpsuy) [reply webpack.config.js]: Ignored

**Review by pradeepnschrodinger — APPROVED** [review]
lgtm

**Pull request merged**